_posts/2023-06-20-introducing-iam-for-openfaas.md

```diff
@@ -0,0 +1,479 @@
+---
+title: "Introducing Identity and Access Management (IAM) for OpenFaaS"
+description: "Granularly control access to the OpenFaaS REST API with IAM for OpenFaaS"
+date: 2023-06-20
+categories:
+- sso
+- authorization
+- iam
+- oauth
+author_staff_member: han
+---
+
+## Get started with IAM for OpenFaaS
+
+In this tutorial we are going to walk you through all the steps required to deploy OpenFaaS with Identity and Access Management (IAM) enabled.
+
```

Comment:
Mention what kind of IdP is compatible, list some examples of ones we've tested.

Reply:
This is listed in the `Configure your identity provider` section.

Reply:
We need the OIDC examples in the intro that's important - I don't mind if it's repeated or just moved here.

Half way down the article feels less intuitive for me to start explaining what kind of IdP can be used.